server/test/unit/plugins/util/test_publish_step.py

```diff
@@ -1180,12 +1180,16 @@ def test_no_available_units(self, mock_get_multiple):
         self.assertEqual(self.step.conduit.save_unit.call_count, 0)
         self.assertEqual(self.step.units_to_download, [])
 
-    def test_calls_get_multiple(self, mock_get_multiple):
-        mock_get_multiple.return_value = []
+    @patch('pulp.plugins.util.publish_step.misc.paginate')
+    def test_calls_get_multiple(self, mock_paginate, mock_get_multiple):
+        """
+        ensure that paginate is used
+        """
+        mock_paginate.return_value = []
 
         self.step.process_main()
 
-        mock_get_multiple.assert_called_once_with('fake_unit_type', [], ['foo'])
+        mock_paginate.assert_called_once_with([], 50)
```

Comment:
Should we assert that self.parent.available_units was specifically used as the first arg to paginate?